src/condor_tools/htcondor_cli/eventlog.py

```diff
@@ -68,40 +91,150 @@ def __init__(self, logger, log_file, **options):
+                else:
+                # print header
+                    print(f"{'Site':<20} {'Evictions':<12} {'Wall Time':<12} {'Good Time':<12} {'CPU Usage':<12} {'Job Starts':<12} {'Job Successes':<14} {'Job Failures':<12}")
+                    # print stats for each site
+                    for site, stats in site_stats.items():
+                        if site:
+                            print(f"{site:<20} {stats['Evictions']:<12} {stats['Wall Time']:<12} {stats['Good Time']:<12} {stats['CPU Usage']:<12} {stats['Starts']:<12} {stats.get('Successes', ''):<14} {stats.get('Failures', ''):<12}")
+        elif options.get("json", True) or options.get("csv", True):
+            # add event summaries as objects to list for JSON output
+            json_output = []
+            for job_id, event_summary in event_summaries.items():
+                # convert cpu usage, wall time, and good time to seconds
+                cpu_usage = event_summary.get("CPU Usage", "0+00:00:00")
+                cpu_usage_seconds = int(cpu_usage.split("+")[1].replace(":", ""))
+                wall_time = event_summary.get("Wall Time", "0+00:00:00")
+                wall_time_seconds = int(wall_time.split("+")[1].replace(":", ""))
+                good_time = event_summary.get("Good Time", "0+00:00:00")
+                good_time_seconds = int(good_time.split("+")[1].replace(":", ""))
+                # replace cpu usage, wall time, and good time with seconds
+                event_summary["CPU Usage"] = cpu_usage_seconds
+                event_summary["Wall Time"] = wall_time_seconds
+                event_summary["Good Time"] = good_time_seconds
+                json_output.append({"Job ID": job_id, **event_summary})
+            if options.get("csv", True):
+                print("Job ID,Host,Start Time,Evict Time,Evictions,Wall Time,Good Time,CPU Usage")
+                for job_id, event_summary in event_summaries.items():
+                    print(f"{job_id},{event_summary.get('Host', '')},{event_summary.get('Start Time', '')},{event_summary.get('Evict Time', '')},{event_summary.get('Evictions', '')},{event_summary.get('Wall Time', '')},{event_summary.get('Good Time', '')},{event_summary.get('CPU Usage', '')}")
+            else:
+                print(json.dumps(json_output, indent=4))
+        else:
+            # determine maximum length of each column
+            max_job_id_len = max(len(str(job_id)) for job_id in event_summaries.keys())+2
+            max_host_len = max(len(data.get("Host", "")) for data in event_summaries.values())+2
+            max_start_time_len = max(len(data.get("Start Time", "")) for data in event_summaries.values())+2
+            max_evict_time_len = max(len(data.get("Evict Time", "")) for data in event_summaries.values())+2
+            max_evictions_len = len(str("Evictions"))+2
+            max_time_format = len(str("0+00:00:00"))+2
```

Comment:
This appears to be a duplicate of code in another branch of the conditional, which almost always means that it should be moved (from both places) to before the conditional, or turned into a function.

Actually, in this case, it looks like you don't use any of this information in this branch of the conditional.  Should this code just be entirely removed?